tests/tpch/test_dask.py

```diff
@@ -785,6 +785,71 @@ def test_query_15(client, dataset_path, fs):
     return result
 
 
+@pytest.mark.shuffle_p2p
+def test_query_16(client, dataset_path, fs):
+    """
+    select
+        p_brand,
+        p_type,
+        p_size,
+        count(distinct ps_suppkey) as supplier_cnt
+    from
+        partsupp,
+        part
+    where
+        p_partkey = ps_partkey
+        and p_brand <> 'Brand#45'
+        and p_type not like 'MEDIUM POLISHED%'
+        and p_size in (49, 14, 23, 45, 19, 3, 36, 9)
+        and ps_suppkey not in (
+            select
+                s_suppkey
+            from
+                supplier
+            where
+                s_comment like '%Customer%Complaints%'
+        )
+    group by
+        p_brand,
+        p_type,
+        p_size
+    order by
+        supplier_cnt desc,
+        p_brand,
+        p_type,
+        p_size
+    """
+    partsupp = dd.read_parquet(dataset_path + "partsupp", filesystem=fs)
+    part = dd.read_parquet(dataset_path + "part", filesystem=fs)
+    supplier = dd.read_parquet(dataset_path + "supplier", filesystem=fs)
+
+    supplier["is_complaint"] = supplier.s_comment.str.contains(
+        "Customer"
+    ) & supplier.s_comment.str.contains("Complaints")
```

Comment:
This is slightly incorrect, it could return `True` for `Complaints Customer`, while the SQL would be `False` 

```suggestion
    supplier["is_complaint"] = supplier.s_comment.str.contains(
        "Customer.*Complaints"
    )
```